src/thresholds.service.js

```diff
@@ -8,29 +11,20 @@ const find = (list, match) => {
   return undefined
 }
 
-const isVersion = (version, item) => item.version === version
-
-const isId = (id, item) => item._id === id
-
-const getFactorVersion = (stockCount, factor, options) => {
-  if (options.version === 'last') {
-    return options.version
-  }
-  if (!(stockCount[factor] && stockCount[factor].version)) {
-    return 1
-  }
-  return stockCount[factor].version
+const isVersion = (date, version) => {
+  const momentDate = moment().isoWeekYear(date.year).isoWeek(date.week).isoWeekday(1).startOf('day')
+  const momentVersionStartDate = moment(version.date, config.versionDateFormat).startOf('isoWeek').startOf('day')
```

Comment:
My intention was to include moment as an ES6 module and then left it out of the bundle with `external` in the rollup.config as we did with Angular, but it doesn't seem to work right now https://github.com/moment/moment/issues/3104, so I reverted to use the global version included in the dashboard. Not the prettiest solution but I can leave with it for now.